src/App.jsx

```diff
@@ -10,14 +10,12 @@ function App() {
     const [country, setCountry] = useState([])
     const [visable, setVisable] = useState([true])
```

Comment:
Bij de initialState heb je een boolean in een array neergezet. Als je een boolean daar wilt neerzetten kan je de array gewoon weglaten. Dus zo: useState(true)